packages/constants/common.ts

```diff
@@ -7,3 +7,7 @@ export const ONE_MINUTE = 60 * ONE_SECOND;
 export const ONE_HOUR = 60 * ONE_MINUTE;
 export const EDIT_DATA_RESOURCES = 'EDIT_DATA_RESOURCES';
 export const DATA_FEDERATION = 'Data Federation';
+export const DEDICATED_ADMIN = 'dedicated-admin';
```

Comment:
r u sure it us  'dedicated-admin' not  'dedicated-admins' ??

Reply:
![image](https://user-images.githubusercontent.com/22158580/177308381-0ce41fab-5ae2-42e1-adab-888bff17cad8.png)

the namespace which got created when I created the OSD cluster